__docs__/wonder-blocks-styles/action-styles-variants.stories.tsx

```diff
@@ -0,0 +1,155 @@
+ */
+export default {
+    title: "Packages / Styles / Action Styles / Action Styles - All Variants",
+    parameters: {
+        docs: {
+            autodocs: false,
+        },
+    },
+    render: () => {
+        const scenarios = [
+            {
+                name: "Using IconButton",
+                props: {
+                    children: (
+                        <View
+                            style={{
+                                flexDirection: "row",
+                                gap: spacing.medium_16,
+                            }}
+                        >
+                            <IconButton
+                                kind="primary"
+                                icon={info}
+                                style={actionStyles.inverse}
+                            />
+                            <IconButton
+                                kind="secondary"
+                                icon={info}
+                                style={actionStyles.inverse}
+                            />
+                            <IconButton
+                                kind="tertiary"
+                                icon={info}
+                                style={actionStyles.inverse}
+                            />
+                        </View>
+                    ),
+                },
+            },
+            {
+                name: "Using Clickable",
+                props: {
+                    children: (
+                        <Clickable
+                            onClick={() => {}}
+                            style={actionStyles.inverse}
+                        >
+                            {() => "Clickable component"}
+                        </Clickable>
+                    ),
+                },
+            },
+            {
+                name: "Using Clickable with initial border",
+                props: {
+                    children: (
+                        <Clickable
+                            onClick={() => {}}
+                            style={[
+                                {
+                                    border: `${border.width.thick}px solid`,
+                                },
+                                actionStyles.inverse,
+                            ]}
+                        >
+                            {() => "Clickable component"}
+                        </Clickable>
+                    ),
+                },
+            },
+            {
+                name: "Using an HTML element",
+                props: {
+                    children: (
+                        <StyledButton
+                            style={[
+                                {
+                                    border: `1px solid ${semanticColor.status.success.background}`,
+                                    backgroundColor:
+                                        semanticColor.status.success.foreground,
```

Comment:
Should `foreground` be used for `color` and `background` used for `backgroundColor`?

Reply:
A comment sounds good to me! So we know it's intentionally swapped 😄